german/the-black-hack.md

```diff
@@ -26,38 +26,39 @@ The Black Hack (TBH) ist ein traditionelles Tischrollenspiel, das mit Papier, St
 
 Wann immer Charaktere versuchen, etwas zu tun, das möglicherweise scheitern könnte, wird der Erfolg mit einer Probe auf einen Eigenschaftswert getestet. Um eine Probe erfolgreich zu bestehen, müssen Spieler mit einem W20 ***unter*** den entsprechenden Eigenschaftswert würfeln.
 
-Monster würfeln keine Proben - Spielercharaktere müssen ihren Angriffen ausweichen, indem sie eine Probe bestehen. Das einzige, was Monster würfeln, ist ihr Schaden.
+Monster würfeln keine Proben - Spielercharaktere müssen ihren Angriffen ausweichen, indem sie eine Probe bestehen. Das einzige, das Monster würfeln, ist ihr Schaden.
 
 ## Eigenschaftswerte auswürfeln
 
-**Eigenschaftswerte** werden mit 3W6 in der folgenden Reihenfolge bestimmt: **Stärke (STÄ), Geschick (GES), Ausdauer (AUS), Intelligenz (INT), Weisheit (WEI), Charisma (CHA)**. Wenn ein gewürfelter Eigenschaftswert 15+ beträgt, muss der folgende mit 2W6 + 2 bestimmt werden, danach wird wieder mit 3W6 gewürfelt, bis eine weitere 15+ fällt. Wenn die Werte für alle Eigenschaften bestimmt wurden, dürfen zwei davon miteinander getauscht werden.
+**Eigenschaftswerte** werden mit 3W6 in der folgenden Reihenfolge bestimmt: **Stärke (STÄ), Geschick (GES), Konstitution (AUS), Intelligenz (INT), Weisheit (WEI), Charisma (CHA)**. Wenn ein gewürfelter Eigenschaftswert 15+ beträgt, muss der folgende mit 2W6 + 2 bestimmt werden, danach wird wieder mit 3W6 gewürfelt, bis eine weitere 15+ fällt. Wenn die Werte für alle Eigenschaften bestimmt wurden, dürfen zwei davon miteinander getauscht werden.
 
 ## Eine Klasse auswählen
 
 Es stehen 4 Klassen zur Auswahl: Krieger, Dieb, Kleriker und Zauberer - die Klasse legt den Schaden fest, den Charaktere bei einem Treffer verursachen, ihre Trefferpunkte und ihre besonderen Fähigkeiten.
 
 ## Ausrüstung kaufen
 
-Neue Charaktere beginnen das Spiel mit 3W6 x 10 Münzen, mit denen sie ihre Ausrüstung kaufen können. Außerdem besitzen sie jeder die Kleidung, die sie tragen und eine Waffe, die ihre Klasse ihnen erlaubt.
+Neue Charaktere beginnen das Spiel mit 3W6 x 10 Münzen, mit denen sie ihre Ausrüstung kaufen können. Außerdem besitzen sie jeder die Kleidung, die sie tragen und eine Waffe, die ihnen ihre Klasse erlaubt.
 
 ## Rüstungspunkte
 
 Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stück Rüstung reduziert Schaden um einen bestimmten Betrag. Charaktere gewinnen **Rüstungspunkte (RP)** zurück, wenn sie sich ausruhen. Wenn Monster oder Charaktere mit ihrer Rüstung Schaden im Gegenwert der Rüstungspunkte aufgefangen haben, sind sie zu erschöpft oder verletzt, um ihre Rüstung weiter effektiv einzusetzen - ab dann nehmen sie vollen Schaden.
 
-|     Typ        | Punkte |
-|:--------------:|:------:|
-| Wams           |   2    |
-| Lederrüstung   |   4    |
-| Kettenhemd     |   6    |
-| Plattenrüstung |   8    |
-| Kleiner Schild |   2    |
-| Großer Schild  |   4    |
+|     Rüstungstyp        | Rüstungspunkte |
```

Comment:
Wir könnten in der Tabelle RP nutzen. Die Abkürzung wird im Text vorher erklärt.